### PR TITLE
feature/SOF-7524 Vacancy in hBN tutorial

### PR DESCRIPTION
**Pull request by VsevolodX**
- feat: add vacncy in hbn tutorial (claude)
- update: add more steps
- update: add images
- update: add a ref and peridos

### CODE REVIEW

File: lang/en/docs/tutorials/materials/specific/defect-point-vacancy-boron-nitride.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+### 2.2. Open and Configure Nanoribbon Notebook.
+
+Find and open `create_nanoribbon.ipynb` in the list of notebooks. Edit the nanoribbon parameters in section 1.1 of the notebook:
+
+```python
+WIDTH = 8  # in number of atoms
+LENGTH = 14  # in number of atoms
+VACUUM_WIDTH = 0  # in number of atoms
+VACUUM_LENGTH = 0 # in number of atoms
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/specific/defect-point-vacancy-boron-nitride.md:55]: How are these distances measured in the number of atoms? Not clear - is this number of layers?

Comment (VsevolodX) [reply lang/en/docs/tutorials/materials/specific/defect-point-vacancy-boron-nitride.md:55]: Yes, layers, but perpendicular to Z-axis

File: lang/en/docs/tutorials/materials/specific/defect-point-vacancy-boron-nitride.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+# Vacancy Point Defects in Hexagonal Boron Nitride.
+
+## Introduction.
+
+This tutorial demonstrates the process of creating materials with vacancy point defects, based on the work presented in the following manuscript:
+
+!!!note "Manuscript"
+    Fabian Bertoldo, Sajid Ali, Simone Manti & Kristian S. Thygesen, "Quantum point defects in 2D materials - the QPOD database", Nature, 2022. [DOI:10.1038/s41524-022-00730-w](https://doi.org/10.1038/s41524-022-00730-w){:target='_blank'}.
+
+We use the [Materials Designer](../../../materials-designer/overview.md) and JupyterLite environment to create a nanoribbon of hexagonal boron nitride (hBN) and introduce vacancy defects. The process combines the capabilities of nanoribbon creation and point defect introduction.
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/specific/defect-point-vacancy-boron-nitride.md:15]: hBN to h-BN

File: mkdocs.yml [hunk 3]
```diff
@@ -222,6 +222,7 @@ nav:
             - Specific:
                 - Substitutional Point Defects in Graphene:                   tutorials/materials/specific/defect-point-substitution-graphene.md
                 - Vacancy-Substitution Pair Defects in GaN:                   tutorials/materials/specific/defect-point-pair-gallium-nitride.md
+                - Vacancy Defect in hBN:                                       tutorials/materials/specific/defect-point-vacancy-boron-nitride.md
```
Comment (timurbazhirov) [inline mkdocs.yml:225]: Vacancy Point Defect

File: lang/en/docs/tutorials/materials/specific/defect-point-vacancy-boron-nitride.md (hunk 1 above)
Comment (timurbazhirov) [reply lang/en/docs/tutorials/materials/specific/defect-point-vacancy-boron-nitride.md:55]: atoms -> unit cells

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**